### DISCUSSION TITLE
[Enhancement] Fix XF absolute layout design

### DISCUSSION THREAD

**Discussion by MhAllan** [dotnet/maui#169] [General]
Currently in XF, let's say you want to achieve this UI: a red frame that have half of it's parent width, and start's from the middle of it's parent:

<img src="https://user-images.githubusercontent.com/13087546/86504806-79258180-be00-11ea-8ac2-98ed828c79f8.PNG" width="300" />

using absolute layout, you think that is the result of this code
```xaml
<Frame BackgroundColor="Red"
	 AbsoluteLayout.LayoutBounds="0.5, 0, 0.5, 0.1"
	 AbsoluteLayout.LayoutFlags="All"/>
```

but in face that will give different result and make the frame centred, the right code will be
```xaml
<Frame BackgroundColor="Red"
	 AbsoluteLayout.LayoutBounds="1, 0, 0.5, 0.1"
	 AbsoluteLayout.LayoutFlags="All"/>
```

I find this weird and not intuitive design, and leads to bad readability. "1, x, 0.5, x" for a code reader should mean the frame is already out of the layout bounds.

how can we put a frame of:  X = 0.4 of the width, Y = 0.6 of height, Width = 0.77 of width, Height = 0.13 of height?

Should be: ```AbsoluteLayout.LayoutBounds="0.4, 0.6, 0.77, 0.13"```  .. easy to read easy to imagine
XF: ```AbsoluteLayout.LayoutBounds="0.81, 0.19, 0.77, 0.13"``` .. not  right

**Comment by MuleaneEve** [comment]
IIRC, the idea behind the current design of AbsoluteLayout was to enable "docking" a child item to any corner...

Still, I agree that the implementation could be rethought to be more intuitive while keeping that feature.
